granule.go

```diff
@@ -6,6 +6,7 @@ import (
 
 	"github.com/apache/arrow/go/v10/arrow"
 	"github.com/google/btree"
+	"github.com/pingcap/errors"
```

Comment:
I think we should just use the golang errors package. Where else do we use this package?